src/styles/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  width: 200px;
+  padding: 15px 14px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: "Roboto", sans-serif;
+
+  &__photo {
+    width: 160px;
+    height: 134px;
+    margin: 15px 5px 42px;
+  }
+
+  &__name {
+    margin: 0;
+    margin-bottom: 5px;
```

Comment:
margin: 0 0 5px;
 you can set values for all sides.
if 4 values - top-right-bottom-left margin: 10px 20px 5px 18px;
if 3 values - top-(left and right)-bottom example: margin: 10px 20px 10px;
if 2 values - (vertical)-(horizontal) example: margin: 10px 20px;